ckc/stripe/utils/subscriptions.py

```diff
@@ -0,0 +1,39 @@
+import stripe
+from djstripe.models import Price, Product
+
+
+def create_price(amount, interval, interval_count=1, currency="usd", product_name="Sample Product Name", **kwargs):
+    """
+    create and return a stripe price object
+    @param amount: the amount to charge
+    @param interval: the interval to charge at
+    @param interval_count: the number of intervals to charge at
+    @param currency: the currency to charge in
+    @param product_name: the name of the product to create
+    @param kwargs: additional arguments to pass to the stripe.Product.create method
+    @returns stripe.Price
+
+    """
+    stripe_product = stripe.Product.create(
```

Comment:
any error handling needed in this function? we can have some fun soon trying to cause all kinds of problems and handle them nicely .. then never have to handle them again!